### PR TITLE
game-tools: initial add-on

### PR DESCRIPTION
**Pull request by lrusak**
Getting this in, not very useful until retroplayer is fully ready, so no rush on merging this.

Could use a proper icon @chewitt when you have time :video_game:

### CODE REVIEW

**Comment by lrusak** [comment]
rebased for review

File: packages/addons/tools/game-tools/changelog.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+8.1.100
```
Comment (CvH) [inline packages/addons/tools/game-tools/changelog.txt]: LE9 or just 100

File: packages/addons/tools/game-tools/package.mk [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+PKG_DEPENDS_TARGET="toolchain"
+PKG_SECTION="virtual"
+PKG_SHORTDESC="A bundle of game tools and programs"
+PKG_LONGDESC="This bundle currently includes bchunk, ecm-tools, iat, and linuxconsoletools"
+PKG_AUTORECONF="no"
+
+PKG_IS_ADDON="yes"
+PKG_ADDON_NAME="Game Tools"
+PKG_ADDON_TYPE="xbmc.python.script"
+PKG_ADDON_PROVIDES=""
```
Comment (CvH) [inline packages/addons/tools/game-tools/package.mk]: not needed if `PKG_ADDON_PROVIDES=""`

File: packages/addons/addon-depends/game-tools/bchunk/package.mk [hunk 3]
```diff
@@ -0,0 +1,36 @@
+################################################################################
+#      This file is part of LibreELEC - http://www.libreelec.tv
+#      Copyright (C) 2016 Team LibreELEC
```
Comment (CvH) [inline packages/addons/addon-depends/game-tools/bchunk/package.mk]: 2017 :) (all other files too)

File: packages/addons/addon-depends/game-tools/bchunk/package.mk [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="bchunk"
+PKG_VERSION="1.2.0"
+PKG_REV="1"
```
Comment (CvH) [inline packages/addons/addon-depends/game-tools/bchunk/package.mk]: we removed `PKG_REV="1"` (all other files too)

**Comment by lrusak** [comment]
updated with @CvH changes

we can do a mass update of the licence headers later

**Pull request merged**